mono/mini/interp/interp.c

```diff
@@ -4621,6 +4622,7 @@ interp_exec_method_full (InterpFrame *frame, ThreadContext *context, FrameClause
 					/* slow path */
 					isinst = mono_object_isinst_checked (o, c, error) != NULL;
 					mono_error_cleanup (error); /* FIXME: don't swallow the error */
+					error_init_reuse (error);
```

Comment:
Same here. could use a local `MonoError`

Reply:
I'm *reducing* stack use here. Many MonoError have already been removed. 100 more bytes will be removing. Adding more MonoError would go in the wrong direction.

Reply:
"here" meaning in other PRs. This PR does not reduce stack.

Reply:
There's already a function call here (`mono_object_isinst_checked`).  Could be `mono_object_isinst_nofail` that sets up a local MonoError.

reusing MonoError is a last resort sort of thing.  There's no reason to do it here.

Reply:
Ignoring errors isn't the right fix either imho. Reusing variables is a normal idiom..
```
s = strup("foo");
free(s)
s = strdup("bar");
```

I know there is functional programming that avoids it, but I've seen very little of that in mono or anywhere and I think for easy efficiency we should be willing to reuse.